applications/volume_rendering_xr/Dockerfile

```diff
@@ -77,23 +47,10 @@ RUN apt update \
         libxrandr-dev \
         libxxf86vm-dev \
         mesa-common-dev \
+        libvulkan-dev="1.3.204.1-*" \
+        glslang-tools="11.8.0+1.3.204.0-*" \
```

Comment:
It might be possible that `glslang-tools` is not needed as well. I don't think the example needs to compile GLSL shaders.

Reply:
Good point, verified that we can remove this package 👍